tests/IO/GeoJSONWriterTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Brick\Geo\Tests\IO;
+
+use Brick\Geo\IO\GeoJSONReader;
+use Brick\Geo\IO\GeoJSONWriter;
+
+class GeoJSONWriterTest extends GeoJSONAbstractTest
+{
+    /**
+     * @dataProvider providerReadGeometry
+     * @dataProvider providerReadFeatureCollection
+     *
+     * @param string $geojson The GeoJSON to read.
+     *
+     * @return void
+     * @throws \Brick\Geo\Exception\CoordinateSystemException
+     * @throws \Brick\Geo\Exception\GeometryIOException
+     * @throws \Brick\Geo\Exception\InvalidGeometryException
+     * @throws \Brick\Geo\Exception\UnexpectedGeometryException
+     */
+    public function testReadGeometry(string $geojson) : void
```

Comment:
This should be called testWriteGeometry()

Reply:
Resolved in commit: 0ac1c49